src/lib/components/input/Input.svelte

```diff
@@ -1,79 +1,94 @@
-					{required}
-					{disabled}
-				/>
-			{:else if type === 'tel'}
-				<input
-					on:input|once={() => (isPristine = false)}
-					{name}
-					type="tel"
-					{placeholder}
-					bind:value
-					class="text-sm focus:outline-none focus:border-0 focus:shadow-none"
-					{required}
-					{disabled}
-				/>
-			{/if}
-		</div>
-	</label>
-	{#if error && !isPristine}
-		<span in:fly={{ x: -12 }} class="text-red-600 text-[12px] ml-3">{error}</span>
-	{:else}
-		<span class="text-red-600 text-[12px] ml-3 invisible" aria-hidden>you haven't read this</span>
-	{/if}
+<div class="flex w-full flex-col items-center">
+  <label
+    class={'flex h-[64px] w-full max-w-[330px] rounded-xl border-4  bg-brand-white pr-4 transition-all ' +
+      className +
+      ' ' +
+      (error && !isPristine ? ' border-[#D14F4F]' : 'border-[#f0f0f0]')}
+  >
+    <div class="flex h-full w-full flex-col justify-center pl-5 text-[15px]">
+      <span class="text-brand-p-black">{label}</span>
+      {#if type === 'text'}
+        <input
+          on:input|once={() => (isPristine = false)}
+          {name}
+          type="text"
+          {placeholder}
+          bind:value
+          class="text-sm focus:border-0 focus:shadow-none focus:outline-none"
+          {required}
+          {disabled}
+        />
+      {:else if type === 'number'}
+        <input
+          on:input|once={() => (isPristine = false)}
+          {name}
+          type="number"
+          {placeholder}
+          bind:value
+          class="text-sm focus:border-0 focus:shadow-none focus:outline-none"
+          {required}
+          {disabled}
+        />
+      {:else if type === 'email'}
+        <input
+          on:input|once={() => (isPristine = false)}
+          {name}
+          type="email"
+          {placeholder}
+          bind:value
+          class="text-sm focus:border-0 focus:shadow-none focus:outline-none"
+          {required}
+          {disabled}
+        />
+      {:else if type === 'tel'}
+        <input
+          on:input|once={() => (isPristine = false)}
+          {name}
+          type="tel"
+          {placeholder}
+          bind:value
+          class="text-sm focus:border-0 focus:shadow-none focus:outline-none"
+          {required}
+          {disabled}
+        />
+      {:else if type === 'date'}
+        <input
+          on:input|once={() => (isPristine = false)}
+          {name}
+          type="date"
+          {placeholder}
+          bind:value
+          class="text-sm focus:border-0 focus:shadow-none focus:outline-none"
+          {required}
+          {disabled}
+        />
+      {/if}
+    </div>
+  </label>
+  {#if error && !isPristine}
+    <strong in:fly={{ x: -12 }} class="ml-3 w-[80%] text-left text-[12px] text-red-600"
+      >{error}</strong
+    >
+  {:else}
+    <strong class="invisible ml-3 w-[80%] text-left text-[12px] text-red-600" aria-hidden
+      >you haven't read this</strong
+    >
```

Comment:
```suggestion
    <strong class="invisible ml-3 w-[80%] text-left text-[12px] text-red-600" aria-hidden
      ></strong
    >
```
Preferible, y ¿por qué el w-[80%]?

Reply:
También hay que buscar la forma de que la animación se ejecute en caso de distintos errores, si es muy complicado/consume mucho tiempo, avisarme para colocarlo como tarea aparte

Reply:
arreglao